api/api.go

```diff
@@ -0,0 +1,93 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/consul-terraform-sync/event"
+)
+
+const (
+	defaultAPIVersion = "v1"
+
+	statusHealthy      = "healthy"
+	statusDegraded     = "degraded"
+	statusCritical     = "critical"
+	statusUndetermined = "undetermined"
```

Comment:
With status being a public API, it would be friendly for us to make these public constants in the event someone creates a client to consume this API, and can you add human readable descriptions on what each status means based on the rolling window of task statuses?

Reply:
Good point about adding a client! Let me add this.

Reply:
Documentation 🥇